Gemfile.lock

```diff
@@ -81,7 +81,7 @@ GEM
     minitest (5.16.2)
     msgpack (1.5.4)
     nio4r (2.5.8)
-    nokogiri (1.13.8-x86_64-darwin)
+    nokogiri (1.13.8-arm64-darwin)
```

Comment:
I don't know what these changes are doing to the gemfile.. but they don't look necessary. I'm going to delete this for now, but we can change this later if we need to!